javatests/arcs/android/storage/handle/AndroidHandleManagerTest.kt

```diff
@@ -113,8 +118,13 @@ class AndroidHandleManagerTest : LifecycleOwner {
         )
     }
 
+    private fun runLifecycleTest(block: suspend CoroutineScope.() -> Unit) = runBlocking {
+        block()
+        lifecycle.setCurrentState(Lifecycle.State.DESTROYED)
```

Comment:
After switching to this lifecycle approach, I forgot to actually move the lifecycle to DESTROYED; I'm hoping this was the issue.

Reply:
(nope)